qemu/qmp.go

```diff
@@ -1033,6 +1052,30 @@ func (q *QMP) ExecQueryMemoryDevices(ctx context.Context) ([]MemoryDevices, erro
 	return memoryDevices, nil
 }
 
+// ExecQueryCpus returns a slice with the list of `CpuInfo`
+// Since qemu 2.12, we have `query-cpus-fast` as a better choice in production
```

Comment:
would be nice to have a function for `query-cpu-fast` might be called `ExecQueryCpusFast` ..

Reply:
OK for me, will do.